misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -90,41 +90,38 @@ abstract class PathAssertionsSpec(
     describeFun(startsWith.name, startsNotWith.name) {
         val startsWithFun = startsWith.lambda
         val startsNotWithFun = startsNotWith.lambda
-        context("starts with") {
-            it("${startsWith.name} does not throw") {
+
+        context("path '/some/path/for/test'") {
+            it("${startsWith.name} '/some/path/' does not throw") {
                 expect(Paths.get("/some/path/for/test"))
```

Comment:
`expect(Paths.get("/some/path/for/test"))` is always the same for the whole context (makes sense, as this is the given context). Therefore I would recommend to extract this to a local variable, you can name it `expectPath` for instance